### PR TITLE
Tella web 1.3.7

### PR DESCRIPTION
**Pull request by carohadad**
@juandans01 @raphmim I wrote here the documentation about the admin center feature!

I wrote about "suspicious login" feature even though we don't have it individualized yet because it is individualized in the contract: https://horizontal-org.slite.com/app/docs/d9juVOhYj-QJYH

I don't like the idea of enabling or disabling EMAILS (in general) . I feel  that's more like an infra-configuration(that we already wrote about in the README about the smtp variables). For exmple in the future if we implement restore password or any other thing that requires email I don't want admins to mistakenly disable "emails" and disable multiple features without knowing what are they are doing.

Could you please review if what I wrote makes sense for now, and once @raphmim review the figma and make a decision on labels I can adapt those.

Could you both please:
- check if the release url is correct or if there is any other place or change that we want to make to the "Install the server" part ? 
- review if the privacy considerations for the features on the admin centar are correct: the goal is for admins to have what they need to make an informed decision about if it's safe for them to enable those in their servers

Gracias!!

### CODE REVIEW

**Comment by juandans01** [comment]
i still think that having the featuree of disabling emails in general is a good idea, in my mind the whole point of an admin center was for people that want to make tellaweb extra secure and *not connect to external services* (also if i remember greenwolf asked specifically to make tellaweb function without emails). 
for the right now i dont think it makes much sense to say "Suspicious login detection" because that is not in the admin center, in any case i think is a good idea to have a list of what features email disabling actually disables (and there put suspicious login).
the release url looks good on my part.

File: docs/For organizations/tella-web.md [hunk 1]
```diff
@@ -57,8 +57,17 @@ Take a look at [this video tutorial](/video-tutorials#connections-full-video) fo
 
 ### Admin center {#admin-center}
 
-Admins of the Tella Web space can change system-wide settings:
+Admins of the Tella Web space can enable or disable system-wide settings depending on their own risk assessment and privacy consideration:
 - [Opt-in to share analytics with Tella team](/security-and-privacy#analytics).
+- Suspicious login detection: 
+    - If enabled Tella Web will record location for each login IP and approximate location based on IP. If the user is connecting from an unusual location Tella Web will block the login attempt to prevent any unauthorized sing-in and send an email to the user to confirm if was a legitimate login attempt. 
+    - IPs and location will not be shared with the Tella team and will only be stored in the server where Tella Web is installed.
```
Comment (juandans01) [inline docs/For organizations/tella-web.md]: this is not true, we use a third party service for understanding which country the ip is in (a service we want to stop using for sure but this is how it is right now)

**Comment by juandans01** [comment]
something we talked with @raphmim today that helps bring context i think.
the idea is to remove the third party service that finds the ip location, so at the end we wouldn't need an admin center variable at all

**Comment by carohadad** [comment]
> something we talked with @raphmim today that helps bring context i think. the idea is to remove the third party service that finds the ip location, so at the end we wouldn't need an admin center variable at all

We are going to to this in the future, but for now, these are the 3 settings on the admin center, correct?

**Review by juandans01 — CHANGES_REQUESTED** [review]

File: docs/For organizations/tella-web.md (hunk 1 above)
Comment (carohadad) [reply docs/For organizations/tella-web.md]: solved!

**Pull request closed without merging**